security/src/main/java/org/aerogear/mobile/security/SecurityCheckMetricPublisher.java

```diff
@@ -35,6 +37,17 @@ public void onExecuted(SecurityCheckResult result) {
     @Override
     public void onComplete() {
         metricsService.publish(SecurityService.SECURITY_METRICS_EVENT_TYPE,
-                        new SecurityCheckResultMetric(metricResults));
+                        new SecurityCheckResultMetric(metricResults))
+                        .respondWith(new Responder<Boolean>() {
```

Comment:
I'm struggling to understand how this fixes the problem. (I don't doubt that it does)
Is it a case of you need to observe the response before it will actually happen? (Quantum HTTP) :)

Reply:
@david-martin Cold Single Observer pattern.  pretty normal Rx stuff